### PR TITLE
modulefiles templates: shell aliases always defined

### PR DESCRIPTION
**Pull request by marcodelapierre**
Signed-off-by: Marco De La Pierre <marco.delapierre@gmail.com>

### CODE REVIEW

**Comment by marcodelapierre** [comment]
Hi @vsoch I have noticed this today, as I was testing the latest release `0.1.18` vs `0.0.57`, which we have been using so far on our supercomputer.
I am proposing to always generate the shell aliases for things like `-shell`, `-container` and so on, even when we use wrapper scripts.
Did you remove them for some specific issue or bug?

From my perspective, I understand It is a tiny inconsistency, but it should not generate big issues; our rationale was that these aliases are by nature mostly used interactively anyway, so should not pose any issue.
On the contrary, not having them can be quite annoying, as it takes out functionality; in particular `-shell` and `-container` can be quite often used. 
In summary, I think reinstating them is the lesser evil.

Eventually, in the future we might provide wrapper scripts for these as well, to be used when wrappers are turned on for the main application commands.

What do you think?

**Comment by vsoch** [comment]
@marcodelapierre I think if wrapper scripts are generated, we generate the aliases but as scripts instead.  The last time we chat about it was for this issue https://github.com/singularityhub/singularity-hpc/issues/565 and I followed up with https://github.com/singularityhub/singularity-hpc/pull/586 that ensured they were generated, e.g., here: https://github.com/singularityhub/singularity-hpc/blob/ee4881a1c505ff3c0d21dbe57e547b1b791d4932/shpc/main/wrappers/__init__.py#L36

So if they aren't in the bin of the module directory, that's probably a bug.

**Comment by marcodelapierre** [comment]
Aaaaargh!! It is really the sign that my weekend is looming...!!

I even tested `-shell`, then forgot, did not find it in the module file, branch out, open the PR ... 🙈
All good, no bug, apart from the weekend bug..!!

Thanks for your patience ;-)
